packages/terra-clinical-result/src/ClinicalResultBloodPressure.jsx

```diff
@@ -9,7 +9,12 @@ import Observation from './common/observation/_Observation';
 import observationPropShape from './proptypes/observationPropTypes';
 import ResultError from './common/other/_ResultError';
 import NoData from './common/other/_KnownNoData';
-import { isEmpty, checkIsStatusInError, ConditionalWrapper } from './common/utils';
+import {
+  isEmpty,
```

Comment:
can we remove isEmpty?

Reply:
Removed all the unused imports https://github.com/cerner/terra-clinical/pull/648/commits/bcd9623974696cc69c750cf77514f41a424ac509#diff-13926e711e02256769a4612c0d911268R13